assets/js/hooks/useWindowSize.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * `useWindowSize` hook.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Copied from https://github.com/jaredLunde/react-hook/blob/b8ac9515e26937e838a36a27001dc46c7f46a390/packages/window-size/throttled/src/index.tsx
+// Modified to use global.innerWidth and global.innerHeight instead of document.documentElement.clientWidth and document.documentElement.clientHeight.
+import { useThrottle } from '@react-hook/throttle';
+import useEvent from '@react-hook/event';
+
+const emptyObj = {};
+
+const win = typeof global === 'undefined' ? null : global;
+const getSize = () => [
+	// document.documentElement.clientWidth,
+	// document.documentElement.clientHeight,
```

Comment:
Commented out code should be avoided. If you want to make it clear that this is a section adapted from the original file you could add a descriptive comment instead but as you have the comment in the header it's probably not needed.

```suggestion
```

Reply:
Addressed [here](https://github.com/google/site-kit-wp/pull/10114/commits/32fe4de2bbdab6fdeac6f85efd1c851931144afc)